content/articles/calculating-fibonacci-numbers-concurrently-in-Python./index.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /calculating-fibonacci-numbers-concurrently-in-Python./
+title: Calculating Fibonacci Numbers Concurrently in Python
+description: This article will cover how to calculate Fibonacci numbers concurrently in Python.
+author:
+date: 2022-05-11T00:00:00-18:30
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/calculating-fibonacci-numbers-concurrently-in-Python./hero.jpg
+    alt: Calculating Fibonacci Numbers Concurrently in Python with examples
+---
+What is so special about Fibonacci numbers? They are numbers used to calculate other mathematical problems. They also have magnetic properties that do not apply to other types of numbers.
+<!--more-->
+One of these properties is recursion used to calculate Fibonacci numbers concurrently in Python using the fastest available algorithms today. What does that mean? You can find the nth Fibonacci number using far fewer computational steps than you think! This article will cover how to calculate Fibonacci numbers concurrently in Python.
+
+### Table of contents
```

Comment:
You've not linked the table of contents correctly. 
It should be done like this ;
[Prerequisites](#prerequisites)
Look at the raw format of this comment to see how.